public/style.css

```diff
@@ -22,4 +22,4 @@ body {
 #review-paragraph {
     max-width: 225px;
 }
-  
\ No newline at end of file
+   */
```

Comment:
Not grabbing a web font anymore?

Reply:
I wasn't sure if I could use both style.css and styled-components together so I cleared style.css

Will do some more testing and figure it out